src/main/java/org/jabref/gui/externalfiles/ImportHandler.java

```diff
@@ -131,10 +131,10 @@ protected List<ImportFilesResultItemViewModel> call() {
                             }
 
                             entriesToAdd.addAll(bibtexParserResult.getDatabaseContext().getEntries());
-                            addResultToList(file, false, Localization.lang("Importing bib entry"));
+                            addResultToList(file, true, Localization.lang("Bib entry was successfully imported"));
```

Comment:
Is there any reason why this second value is now "true"? I am probably msissing some context here, but I was just wondering

Reply:
Ah thanks a lot!

Reply:
Hi @Siedlerchr, sorry I just saw your comment. Thanks for changing the CHANGELOG and approving my changes.